pkg/operator/configobserver/etcd/observe_etcd_endpoints.go

```diff
@@ -106,6 +106,13 @@ func innerObserveStorageURLs(fallbackObserver fallBackObserverFn, alwaysAppendLo
 			errs = append(errs, ipErr)
 			continue
 		}
+
+		// explicitly report reserved IPs to the user
+		if strings.HasPrefix(ip.String(), "192.0.2.") || strings.HasPrefix(ip.String(), "2001:db8:") {
+			errs = append(errs, fmt.Errorf("configmaps/%s in the %s namespace: %v is reserved, please use a different address", etcdEndpointName, EtcdEndpointNamespace, ip.String()))
```

Comment:
optional change.

`configmap %s/%s: address %v is reserved and not supported`

Reply:
done